### ISSUE TITLE
At any time, only one object should be visually indicated to be the focus

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#9632]
### Bug description

At any time, only one object can be the focus, and so only one object should be visually indicated to be the focus. Having more than one object appearing to be the focus is confusing and non-standard.

### Steps to reproduce

Have a project with one or more audio tracks.

Case A:
Tab to the control panel of a track

Case B:
Tab to the toolbar.

### Expected behavior

Case A:
Only the track control panel is visually indicated to be the focus.

Case B:
Only an item on the toolbar is visually indicated to be the focus.

### Actual behavior

For both Case A and B, the track is also visually indicated to be the focus.

### Audacity Version

d2f1347

### Operating system

Windows
